internal/builders/go/pkg/build_test.go

```diff
@@ -1047,3 +1048,114 @@ func Test_generateCommand(t *testing.T) {
 func asPointer(s string) *string {
 	return &s
 }
+
+func TestGoBuild_Run(t *testing.T) {
+	type fields struct {
+		cfg     *GoReleaserConfig
+		goc     string
+		argEnv  map[string]string
+		ldflags string
+	}
+	type args struct {
+		dry bool
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		wantErr bool
```

Comment:
Could you use the expected error `error err` instead of `bool`?

Reply:
I also included the error

Reply:
you should be able to remove the `wantErr` altogether as well?